sormas-backend/src/main/java/de/symeda/sormas/backend/util/ModelConstants.java

```diff
@@ -23,4 +23,9 @@
 	String PERSISTENCE_UNIT_NAME_AUDITLOG = "auditlogPU";
 	String HINT_HIBERNATE_READ_ONLY = "org.hibernate.readOnly";
 
+	/**
+	 * Hard limit how much parameter can be allowed in a query.<br />
```

Comment:
how much parameter => how many parameters
to big => too big
SQL-Statement => SQL statement

Reply:
fixed